### ISSUE TITLE
homebridge-lgwebos-tv won't work in UK & Europe

### ISSUE THREAD

**Issue by jantimon** [grzegorz914/homebridge-lgwebos-tv#98]
from: https://github.com/hobbyquaker/lgtv2/issues/13#issuecomment-351416436

>  if you're in the UK (potentially anywhere in Europe, actually), for whatever ridiculous regulatory reason, TVs are not allowed to stay connected to a network while in standby.

I tried reading the current power state:

```
lgtv.subscribe('ssap://com.webos.service.tvpower/power/getPowerState', (error, response) => {
      console.log({error, response});
  });  
```

and received `{ error: null, response: {} }` - so no error but also an empty result.

do you have any idea how I could work around this?

**Comment by grzegorz914** [comment]
This is not true, I'm in Europe and all my TVs still connected to WiFi network in standby.

Probably Your TV not support this feature, what a model TV U have?

**Comment by jantimon** [comment]
Thanks for looking into it :) I am using the LG OLED 65EF950V

Right now I am using https://github.com/luisiam/homebridge-cmdswitch2#readme to turn it off using a small lgtv2 based node script
